### ISSUE TITLE
Replace diff line modified/deleted/ inserted audio cues with punchier, more distinct ones

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#175014]
@jooyoungseo has indicated that the diff inserted / modified audio cues are hard to differentiate and that the sounds are not as fast / singular toned as they should be

**Comment by jvesouza** [comment]
> @jvesouza what do you think of this?

I agree with the suggestion. At least for me it is very difficult to identify the difference between the sounds used.

**Comment by AFre100** [comment]
Sounds good -- will start on this asap.

**Comment by rzhao271** [comment]
I don't hear the cues as often anymore.
I also noticed that when using up/down keys on the right diff editor, no audio cue shows up for a deleted section.

Example with .gitignore file:

Left:
```
node_modules
__pycache__
.venv/
.venv1/
.vscode/.venv2
pyenv4
.venv2
build
```

Right:
```
node_modules
__pycache__
.vscode/.venv2
pyenv4
.venv*
build
```

**Comment by meganrogge** [comment]
@rzhao271 there are only modifications and insertions on the right side of the diff view because deletions are displayed only on the left side

**Comment by meganrogge** [comment]
> I don't hear the cues as often anymore.

I can reproduce

**Comment by meganrogge** [comment]
@hediet any idea what could be going on here? It's reaching the line I expect it to but there's no sound sometimes
https://user-images.githubusercontent.com/29464607/234720319-525fdb55-57c3-4e60-bfa0-232c8e2678bd.mov

**Comment by meganrogge** [comment]
@hediet it's getting to the line I expect it to (see logs) but no sound is produced - thoughts?

https://user-images.githubusercontent.com/29464607/234721239-26879b16-18bf-4412-89db-0ab5a6cc2c58.mov
